### DISCUSSION TITLE
how can i use my own svg image from asset folder in the ng zorro menu

### DISCUSSION THREAD

**Discussion by Tanujasharma** [NG-ZORRO/ng-zorro-antd#7847] [General]
currently i am using nzicon in my menu but now i need to use my own icons from asset folder in recursive ng zorro menu and submenu 
![image](https://user-images.githubusercontent.com/84020250/221112473-f61ae755-604c-4aba-8618-272d5e356fee.png)
 how can i add the icon from asset folder to menu

**Comment by Edgar-Chmylko-WE** [comment]
Hi, there are 2 ways : 
1. Static way adding icons one by one.
![image](https://user-images.githubusercontent.com/117636642/221126745-3e7dda0e-413a-441b-8dfe-f3a7465fcef5.png)
2. Dynamic way adding your assets folder to angular.json file 
![image](https://user-images.githubusercontent.com/117636642/221126860-ce10a5db-1c8d-4bdc-95a2-9c323ba41b06.png)

**Comment by Edgar-Chmylko-WE** [comment]
Some additional information 
![image](https://user-images.githubusercontent.com/117636642/221127030-946514c9-09c7-4a74-be17-157993f97f89.png)

**Reply by webstermobile** [reply]
这个才是最好用的
